site/src/api/errors.ts

```diff
@@ -71,3 +71,9 @@ export const getErrorMessage = (
     : error instanceof Error
     ? error.message
     : defaultMessage
+
+export const getValidationErrorMessage = (error: Error | ApiError | unknown): string => {
+  const validationErrors =
+    isApiError(error) && error.response.data.validations ? error.response.data.validations : []
+  return validationErrors.map((error) => error.detail).join("\n")
+}
```

Comment:
I think some function level commenting here might be useful in differentiating this method from other similarly named methods in this file.

Reply:
Added it!